src/client/js/partials/email-preview.js

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const emailsPartial = document.querySelector("[data-partial='emailPreview']")
+let emailPreviewForm
+let emailTemplateSelect
+
+function init () {
+  emailPreviewForm = document.querySelector('.js-email-preview-form')
+  emailTemplateSelect = emailsPartial.querySelector('.js-email custom-select')
+
+  emailPreviewForm.addEventListener('submit', sendTestEmail)
+  emailTemplateSelect.addEventListener('change', handleEvent)
+}
+
+function handleEvent (event) {
+  const templateSelectChanged = event.target.matches(
+    'custom-select[name="email-template"]'
+  )
+
+  if (templateSelectChanged) {
+    const updatedPath = `/admin/emails/${event.target.value}`
+    if (window.location.pathname !== updatedPath) {
+      window.location.replace(updatedPath)
+    }
+  }
+}
+
+async function sendTestEmail (event) {
+  event.preventDefault()
+  const selectedRecipient = event.target.querySelector(
+    'input[name="email-recipient-option"]:checked'
+  ).value
+
+  try {
+    const csrfToken = document
+      .querySelector('.js-email[data-csrf-token]')
+      .getAttribute('data-csrf-token')
+
+    const response = await fetch('/admin/send-test-email', {
+      headers: {
+        'Content-Type': 'application/json',
+        'x-csrf-token': csrfToken
+      },
+      mode: 'same-origin',
+      method: 'POST',
+      body: JSON.stringify({
+        emailId: emailTemplateSelect.value,
+        recipient: selectedRecipient
+      })
+    })
+
+    if (response?.redirected) {
+      throw response.error
+    }
```

Comment:
suggestion (non-blocking): should we let the user know if/when email was sent successfully?  A simple browser alert could suffice:
```suggestion
    }
    window.alert('Email sent!')
```

Reply:
That’s a good idea until we have time to make things more user-friendly — [c8f3527](https://github.com/mozilla/blurts-server/pull/2790/commits/c8f3527e75a27a7a58636de5a0cb019f0c3a9aa0).